testing/src/test/java/io/stargate/it/http/graphql/graphfirst/SchemaDeploymentTest.java

```diff
@@ -36,6 +39,8 @@
 @ExtendWith(CqlSessionExtension.class)
 public class SchemaDeploymentTest extends BaseOsgiIntegrationTest {
 
+  private static final int NUMBER_OF_RETAINED_SCHEMA_VERSIONS = 10;
```

Comment:
we don't have access to `SchemaSourceDao#NUMBER_OF_RETAINED_SCHEMA_VERSIONS` so I duplicated it here. 
I am not sure how can we handle it without the needed duplication :/

Reply:
Yeah I think it's acceptable to have a bit of duplication in ITs, we've done it in a few places already.